src/Application/Console/Formatters/Console.php

```diff
@@ -336,7 +336,26 @@ private function issues(
                     }
 
                     if ($detail->hasMessage()) {
-                        $detailString .= ($detailString !== '' ? ': ' : '') . $detail->getMessage();
+                        if ($detail->hasDiff()) {
+                            $hasColor = false;
+                            foreach (explode(PHP_EOL, $detail->getMessage()) as $line) {
+                                if (mb_strpos($line, '-') === 0) {
+                                    $hasColor = true;
+                                    $detailString .= '<fg=red>';
+                                }
+                                if (mb_strpos($line, '+') === 0) {
+                                    $hasColor = true;
+                                    $detailString .= '<fg=green>';
+                                }
+                                $detailString .= $line . PHP_EOL;
+                                if ($hasColor) {
+                                    $hasColor = false;
+                                    $detailString .= '</>';
+                                }
+                            }
+                        } else {
+                            $detailString .= ($detailString !== '' ? ': ' : '') . $detail->getMessage();
+                        }
```

Comment:
Maybe we should extract this logic into a function like `parseDetailMessage(Detail $detail)`

Then, 
```
if ($detail->hasMessage()) {
    $detailString .= ($detailString !== '' ? ': ' : '') . $this->parseDetailMessage($detail)
}
```
Because actually, we lost this part `($detailString !== '' ? ': ' : '')` when it's a diff.

```php
private function parseDetailMessage(Detail $detail): string
{
	if (! $detail->hasDiff()) {
        return $detail->getMessage();
    }
    $message = '';
    $hasColor = false;
    foreach (explode(PHP_EOL, $detail->getMessage()) as $line) {
        if (mb_strpos($line, '-') === 0) {
            $hasColor = true;
            $message .= '<fg=red>';
        }
        if (mb_strpos($line, '+') === 0) {
            $hasColor = true;
            $message .= '<fg=green>';
        }
        $message .= $line . PHP_EOL;
        if ($hasColor) {
            $hasColor = false;
            $message .= '</>';
        }
    }
    
    return $message;
}

```

WDYT ?

Reply:
Good idea. Made an implementation, was it like that you wanted it?